test_assets/TestAssetsForSemanticFeedback_9-Oct-2023.tsv

```diff
@@ -0,0 +1,59 @@
+AssetIdentifier	Query	Settings	InputName (user choice)	InputID, node normalized	OutputID	OutputName	Expected Output	Persona	Well Known	GitHubIssue	Source?

```

Comment:
Is this the finalized schema for assets?

Reply:
Nope
Just chasing after Jenn, keeping her efforts visible to all of us.
We can iterate in anger over the coming days and weeks, to make swift progress.
(I apologize that I've not made so much progress myself due to other professional and non-work distractions... Going to push hard this week to make more progress)